fastembed/embedding.py

```diff
@@ -468,18 +460,18 @@ def __init__(
         self.model_name = model_name
 
         if cache_dir is None:
-            cache_dir = Path(".").resolve() / "local_cache"
+            default_cache_dir = Path(".").resolve() / "local_cache"
+            cache_dir = Path(os.getenv("FASTEMBED_CACHE_PATH", default_cache_dir))
```

Comment:
Change this to tmp directory from current directory please? As mentioned in the issue #50 

https://docs.python.org/3/library/tempfile.html

Reply:
Do we not want the models to persist? Also, the `/tmp` recommended in the issue wouldn't persist across reboots. So I kept the fallback path to the current dir as before.

Reply:
Yes, that's the intent — it shouldn't persist across reboots by default. The OS ENV options means someone can choose to persist them, but by default — it shouldn't. 

This is partially because we want to treat Serverless e.g. AWS Lambda as well as we can. This allows someone to use Qdrant from the AWS Marketplace and the client as serverless for both ingestion and querying

Reply:
Wouldn't it be better if the users on serverless systems set it to `/tmp` themselves? As most other users would expect persistence, I think.

Reply:
As of now, of the 7-10 users I've spoken to, most are using serverless — hence the reasoning

Reply:
https://github.com/qdrant/fastembed/pull/68/commits/0b671bc4e5501cb06c1da9e492300c953d9311d0.